lib/math/vector2d.js

```diff
@@ -57,9 +57,18 @@ export function flip(v, b) {
   return sub(v, scale(sub(v, vOnB), 2));
 }
 
+// CW, of all things.
```

Comment:
wut

Reply:
I wrote this for asteroids, which uses left-hand coordinates, and I didn't notice that I had gotten the rotation backwards. Oops.